index.js

```diff
@@ -1325,6 +1313,12 @@ Renderer.prototype.draw = function() {
             uFogDensity: postProcessingCmp.fogDensity,
             uSunPosition: postProcessingCmp.sunPosition,
             uOutputEncoding: ctx.Encoding.Gamma,
+            uOverlay:
```

Comment:
`postProcessingCmp && postProcessingCmp.enabled ` don't need to be checked, you are already in this condition